katas/main/src/pokerhands/juan/PokerHands.cs

```diff
@@ -0,0 +1,437 @@
+﻿namespace PokerHands.Juan
```

Comment:
Please try to separate some of the classes in separate files for easier navigation through the code